### PR TITLE
Switch flutter run_mojo over to flutter.mojo

### PR DESCRIPTION
**Pull request by abarth**
The new flutter.mojo is ready for use.

### CODE REVIEW

**Comment by abarth** [comment]
@jamesr @krisgiesing

File: packages/flutter_tools/lib/src/artifacts.dart [hunk 1]
```diff
@@ -121,17 +121,17 @@ class ArtifactStore {
       hostPlatform: HostPlatform.mac
     ),
     const Artifact._(
-      name: 'Sky Viewer',
-      fileName: 'sky_viewer.mojo',
-      category: _kViewerCategory,
-      type: ArtifactType.viewer,
+      name: 'Flutter for Mojo',
+      fileName: 'flutter.mojo',
+      category: _kShellCategory,
```
Comment (jamesr) [inline packages/flutter_tools/lib/src/artifacts.dart]: interesting taxonomy here

**Comment by jamesr** [comment]
lgtm

File: packages/flutter_tools/lib/src/artifacts.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter_tools/lib/src/artifacts.dart]: Yeah, the category thing was a mistake.  I've been moving everything over to the shell category with the intent of killing categories whenever we change buckets.

**Pull request merged**